debian/libeos-knowledge-content-0-0.install

```diff
@@ -0,0 +1,2 @@
+usr/lib/libeos-knowledge-content-0.so.*
+usr/lib/gio/modules/libeknvfs.so
```

Comment:
@mattdangerw yes we do need to call gio-querymodules in a post install hook
Its needed to update the cache of extension points, that way gio does not have to open the module each time to know which extension point it implements